integrations/zopim/lib/index.js

```diff
@@ -61,7 +61,7 @@ Zopim.prototype.load = function(done) {
   var scriptUrl = fmt('//v2.zopim.com/?%s', zopimId);
 
   /* eslint-disable */
-  window.$zopim||(function(d,s){var z=$zopim=function(c){z._.push(c)},$=z.s=d.createElement(s),e=d.getElementsByTagName(s)[0];z.set=function(o){z.set._.push(o)};z._=[];z.set._=[];$.async=!0;$.setAttribute('charset','utf-8');$.src=scriptUrl;z.t=+new Date;$.type='text/javascript';e.parentNode.insertBefore($,e)})(document,'script');
+  window.$zopim || (function(d, s) { var z = (window.$zopim = function(c) { z._.push(c); }), $ = (z.s = d.createElement(s)), e = d.getElementsByTagName(s)[0]; z.set = function(o) { z.set._.push(o); }; z._ = []; z.set._ = []; $.async = !0; $.setAttribute('charset', 'utf-8'); $.src = scriptUrl; z.t = +new Date(); $.type = 'text/javascript'; e.parentNode.insertBefore($, e); })(document, 'script');
```

Comment:
the compiled code for zopim would break because of a bad assignment.
The only real change here is `$zopim=` to `window.$zompim=`